components/Results/ResultSettingArea.tsx

```diff
@@ -0,0 +1,90 @@
+import { FunctionComponent } from "react";
+import BackButton from "../../components/Board/HUD/BackButton";
+import { roomAtom } from "../../atoms/room";
+import { useAtomValue } from "jotai/utils";
+import styled from "styled-components";
+
+interface Props {}
+
+const ResultSettingArea: FunctionComponent<Props> = ({}) => {
+  const room = useAtomValue(roomAtom);
+  const teams = room.teams;
+  const spectators = teams[0];
+  let activeTeams: { teamNumber: number; score: number }[] = [];
+  teams.map((item, index) => {
+    if (index > 0) {
+      const team = { teamNumber: index, score: item.score };
+      activeTeams.push(team);
+    }
+  });
```

Comment:
Using `map` works but is misleading, `foreach()` is what you are looking for. This works but let's change it to follow JS paradigms. You can do `teams.filter()` to filter out spectators. Then `map` to follow the form you want, `map` returns an array in the form you want so it would be `const activeTeams = teams.map()`. It just happens that both can be chained so the end result would be:
`const activeTeams = teams.filter((_item, index)=> index!==0).map(item=>({teamNumber: index, score: item.score}));`